  0%|                                                                                           | 0/2000 [00:00<?, ?it/s]/home/dogoulis/anaconda3/lib/python3.8/site-packages/torch/nn/functional.py:718: UserWarning: Named tensors and all their associated APIs are an experimental feature and subject to change. Please do not use them for anything important until they are released as stable. (Triggered internally at  /opt/conda/conda-bld/pytorch_1623448278899/work/c10/core/TensorImpl.h:1156.)
  return torch.max_pool2d(input, kernel_size, stride, padding, dilation, ceil_mode)
  0%|▏                                                                                  | 6/2000 [00:00<03:05, 10.74it/s]














































 99%|███████████████████████████████████████████████████████████████████████████████▎| 1983/2000 [01:33<00:00, 23.94it/s]
100%|████████████████████████████████████████████████████████████████████████████████| 2000/2000 [01:33<00:00, 21.28it/s]
Traceback (most recent call last):
  File "evaluation.py", line 191, in <module>
    main()
  File "evaluation.py", line 184, in main
    log_metrics(y_true=y_true, y_pred=y_pred)
  File "evaluation.py", line 107, in log_metrics
    test_acc = accuracy_score(y_true, y_pred)
  File "/home/dogoulis/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 185, in accuracy_score
    y_type, y_true, y_pred = _check_targets(y_true, y_pred)
  File "/home/dogoulis/anaconda3/lib/python3.8/site-packages/sklearn/metrics/_classification.py", line 97, in _check_targets
    raise ValueError("{0} is not supported".format(y_type))
ValueError: unknown is not supported